### ISSUE TITLE
Expand section on scheduling in additional.html to include length and weekend considerations

### ISSUE THREAD

**Issue by cameronneylon** [sparcopen/opencon-dei-report#11]
In applying the checklist to a FSCI self assessment I noted that two potentials criticism that aren't captured are: 1) the question of whether holding a meeting over a weekend is exclusionary and 2) considerations of the length of events.

For FSCI the length (five days) raised concerns for some and in the past I've frequently been engaged in discussions about those who are excluded by holding an event during the work week (those who volunteer effort in their spare time and can't leave work) or during the weekend (those who work on the topic and shouldn't be expected to take additional time out). There aren't any easy solutions here but raising those issues might be valuable.

This would involve: 1) adding some content to additional.html to address these issues and 2) potentially expanding checklist.html under "Choosing a Date" to explicitly consider the balance of risks for these issues.

I'd be happy to write these sections and submit a pull request but wanted to check in and see if any discussion was merited before doing that.

**Comment by gedankenstuecke** [comment]
The timing is a very good hint! We discussed this at length at this year's [Bioinformatics Open Source Conference](https://www.open-bio.org/wiki/BOSC_2017), as our conference ran over the weekend. In addition to the points you already mentioned much of our discussion focused around how it impacts people who have families/kids. Basically two positions emerged at the end: 

For some people it's easier if events run over the weekend, as this means that organizing childcare is easier, as partners, grandparents et al. are having an easier time to look after children while one is absent for the conference. Other people felt that they have to organize external childcare in any case (i.e. pay for it) and that it means that they are missing out on family time during the weekend.

Similar to the already mentioned points there's no easy solution to this conundrum (though having on-site childcare helps to alleviate the problem a bit), so it's probably best to think about the audience that one wants to target with an event and make the trade-offs accordingly.

**Comment by JosephMcArthur** [comment]
We've certainly discussed these trade-offs before and I personally would support including this. I also don't know of any solutions here either, I think it's just about being aware & deliberate about trade-offs.

**Comment by cameronneylon** [comment]
Yes, that would be my approach. Flag the issue for consideration and say that its tough to resolve but should be thought through and discussed with those affected (we ended up scheduling a bunch of meetings Friday/Saturday, which kind of helps and kind of really doesn't)
